x-pack/plugins/monitoring/public/views/license/controller.js

```diff
@@ -54,12 +54,14 @@ export class LicenseViewController {
         expiryDate = formatDateTimeLocal(license.expiry_date);
       }
 
+      const capitalizedType = license.type[0].toUpperCase() + license.type.slice(1);
```

Comment:
This does fix the issue, but I think that this is the wrong place to fix it. We have two examples of this component being used and it's already been used differently between the two.

Instead of fixing it at the caller level, I think we should just do the case management of the license type inside of the component, which eliminates the need to think about it _and_ simplifies caller code.

https://github.com/elastic/kibana/blob/d42fef6752837c1b84f8eca9f4a0e9de807ae7e0/x-pack/plugins/license_management/public/sections/license_dashboard/license_status/license_status.js#L20

If you modify `type` there to have this behavior (you will also want to explicitly lowercase the `type.slice(1)` portion for parity; you should probably also use `type.substr(1)` like the other code), then this code change can go away _and_ the code that you're effectively copying can also be removed:

https://github.com/elastic/kibana/blob/32505afd3d74df403bc51eb6cf3269e92f84ab94/x-pack/plugins/license_management/public/sections/license_dashboard/license_status/license_status.container.js#L13

You may also be interested in Lodash's [`startCase` function](https://lodash.com/docs/3.10.1#startCase).